axelor-production/src/main/java/com/axelor/apps/production/service/configurator/ConfiguratorProdProcessLineServiceImpl.java

```diff
@@ -38,14 +42,52 @@ public ConfiguratorProdProcessLineServiceImpl(ConfiguratorService configuratorSe
   public ProdProcessLine generateProdProcessLine(
       ConfiguratorProdProcessLine confProdProcessLine, JsonContext attributes)
       throws AxelorException {
-    if (confProdProcessLine == null) {
+    if (confProdProcessLine == null || !checkConditions(confProdProcessLine, attributes)) {
       return null;
     }
+
+    String name;
+    Integer priority;
+    StockLocation stockLocation;
     ProdProcessLine prodProcessLine = new ProdProcessLine();
     BigDecimal minCapacityPerCycle;
     BigDecimal maxCapacityPerCycle;
     long durationPerCycle;
 
+    if (confProdProcessLine.getDefNameAsFormula()) {
+      Object computedName =
+          configuratorService.computeFormula(confProdProcessLine.getNameFormula(), attributes);
+      if (computedName == null) {
+        throw new AxelorException(
+            confProdProcessLine,
+            TraceBackRepository.CATEGORY_INCONSISTENCY,
+            I18n.get(
+                String.format(
+                    IExceptionMessage.CONFIGURATOR_PROD_PROCESS_LINE_INCONSISTENT_NAME_FORMULA,
+                    confProdProcessLine.getId())));
+      } else {
+        name = String.valueOf(computedName);
+      }
+    } else {
+      name = confProdProcessLine.getName();
+    }
+    if (confProdProcessLine.getDefPriorityAsFormula()) {
+      priority =
+          new Integer(
```

Comment:
If I fill a formula returning null for the priority, a `NumberFormatException` will be thrown here. I think we can use 0 as priority if the value returned by the script is null.